app/src/pages/inside/stepPage/modals/testItemDetailsModal/testItemDetailsModal.jsx

```diff
@@ -295,9 +295,13 @@ export class TestItemDetailsModal extends Component {
     const {
       data: { item, eventsInfo },
     } = this.props;
+    const customEventsInfo = {
+      ...eventsInfo,
+      onOpenStackTraceEvent: () => eventsInfo.clickExpandEvent,
```

Comment:
looks like you can add this event directly from stepPage or uniqueErrorsPage with right naming.
Also please remove L153-L155 properties in this file, they looks unnecessary and `ContainerWithTabs` component soesn't use it